Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/Assets/Scripts/AppState.cs

```diff
@@ -0,0 +1,256 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using System.Collections;
+using HoloToolkit.Unity;
+using System;
+using UnityEngine.VR.WSA.Input;
+
+public class AppState : Singleton<AppState>
+{
+    // Consts
+    public float kMinAreaForStats = 5.0f;
+    public float kMinAreaForComplete = 50.0f;
+    public float kMinHorizAreaForComplete = 25.0f;
+    public float kMinWallAreaForComplete = 10.0f;
+
+    // Config
+    public TextMesh DebugDisplay;
+    public TextMesh DebugSubDisplay;
+    public Transform Parent_Scene;
+    public SpatialMappingObserver MappingObserver;
+    public Cursor AppCursor;
+
+    // Properties
+    public string SpaceQueryDescription
```

Comment:
NIT: I wonder if you can get away with public string SpaceQueryDescription { get; set; }
Unless you really need objectPlacementDescription = "";